tests/testcase.mk

```diff
@@ -1,6 +1,9 @@
+ifeq ($(TESTSROOTDIR),)
```

Comment:
Maybe it's time we went through all the tests and changed it to include ../testcase.mk?